English_Documentation/TRON_Protocol/TRON_Wallet_RPC-API.md

```diff
@@ -59,183 +59,307 @@ Transfer, vote, issuance of token, or participation in token offering. Sending s
-14.5 Function List of all issued tokens. Display of all issued tokens for user’s reference.
+14.1 Interface statement
+rpc GetAssetIssueList (EmptyMessage) returns (AssetIssueList) {};  
+14.2 Node
+Fullnode and soliditynode.  
+14.3 Parameters
+EmptyMessage: null.  
+14.4 Returns
+AssetIssueList: AssetIssueContract list containing information on all issued tokens.  
+14.5 Function
+Query of list of all issued tokens. Display of all issued tokens for user’s reference.
 
 ## 15. Query of tokens issued by a given account
-15.1 Interface statement rpc GetAssetIssueByAccount (Account) returns (AssetIssueList) {};   
-15.2 Nodes Fullnode and soliditynode. 15.3 Parameters Account: address.   
-15.4 Returns AssetIssueList: AssetIssueContract list containing information on all issued tokens.   
-15.5 Function Query of all tokens issued by a given account.
+15.1 Interface statement
+rpc GetAssetIssueByAccount (Account) returns (AssetIssueList) {};  
+15.2 Nodes
+Fullnode and soliditynode.
+15.3 Parameters
+Account: address.  
+15.4 Returns
+AssetIssueList: AssetIssueContract list containing information on all issued tokens.  
+15.5 Function
+Query of all tokens issued by a given account.
 
 ## 16. Query of token information by token name
-16.1 Interface statement rpc GetAssetIssueByName (BytesMessage) returns (AssetIssueContract) {};  
- 16.2 Nodes Fullnode and soliditynode. 16.3 Parameters BytesMessage: token name.  
- 16.4 Returns AssetIssueContract: information on the token.  
- 17.5 Function Query of token information with the name. The exclusiveness of token name is ensured on TRON’s network.
+16.1 Interface statement
+rpc GetAssetIssueByName (BytesMessage) returns (AssetIssueContract) {};  
+16.2 Nodes
+Fullnode and soliditynode.
+16.3 Parameters
+BytesMessage: token name.  
+16.4 Returns
+AssetIssueContract: information on the token.  
+16.5 Function
+Query of token information with the name. The exclusiveness of token name is ensured on TRON’s network.
 
 ## 17. Query of current tokens by timestamp
-17.1 Interface statement rpc GetAssetIssueListByTimestamp (NumberMessage) returns (AssetIssueList){};  
-17.2 Node Soliditynode.   
-17.3 Parameters NumberMessage: current timestamp (the number of milliseconds since 1970) .  
-17.4 Returns AssetIssueList: AssetIssueContract list including detailed information of the tokens.   
-17.5 Function List of issued tokens by timestamp. Display of current nodes for users’ reference.
+17.1 Interface statement
+rpc GetAssetIssueListByTimestamp (NumberMessage) returns (AssetIssueList){};  
+17.2 Node
+Soliditynode.  
+17.3 Parameters
+NumberMessage: current timestamp (the number of milliseconds since 1970).  
+17.4 Returns
+AssetIssueList: AssetIssueContract list including detailed information of the tokens.
+17.5 Function
+Query of list of issued tokens by timestamList of issued tokens by timestamp. Display of current nodes for users’ reference.
```

Comment:
To make it clear, add `query of` in the description.

Reply:
I think this line is wrong, should be:

Query list of issued tokens by timestamp. Display of current nodes for users’ reference.

Reply:
it looks better